icechunk-python/tests/test_virtual_ref.py

```diff
@@ -414,3 +414,50 @@ def test_error_on_non_authorized_virtual_chunk_container() -> None:
 
     with pytest.raises(IcechunkError, match="file:///foo.*authorize"):
         array[0]
+
+
+def test_cannot_write_invalid_urls() -> None:
```

Comment:
The traceback I get running our reproducer is
```
icechunk.IcechunkError:   × error parsing virtual ref URL
  │
  │ context:
  │
  │
  ├─▶ error parsing virtual ref URL
  ╰─▶ relative URL without a base
```

Can we print the url here?

Reply:
huh, w t f

Reply:
ah ok ... the unherlying error is printed, but not the url, that's my bad ... fixing

Reply:
in python this would `{url!r}` :)